### DISCUSSION TITLE
Error in Dashboard widgets of pending and active chats

### DISCUSSION THREAD

**Discussion by Advertalis** [LiveHelperChat/livehelperchat#2070] [General]
Hello!
Since the last updates I got a strange behavior of widgets that display active and pending chats.
Right after the icons, instead of the number I see this code:

{{pending_chats.list.length != false && pending_chats.list.length > 0 ? ' ('+pending_chats.list.length+')' : ''}}

{{active_chats.list.length != false && active_chats.list.length > 0 ? ' ('+active_chats.list.length+')' : ''}}

If I click one of them, I can see the full widget with correct visualization.
What happened? 
How can I fix it?

Thanks,
Alan

![2024-03-20_095151](https://github.com/LiveHelperChat/livehelperchat/assets/1754049/030b7883-10aa-4227-9ae9-36e4381c6ac5)

**Comment by remdex** [comment]
Will be fixed in 4.34v. In general I suggest just edit dashboard settings and have only main dashboard there :)
